bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,22 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  object Owned {
```

Comment:
I don't believe that'll work anyway -- You could though add them in as AnyVal case classes here, and make the injections be to those. It would be a similar strategy as to how the Base64String stuff is done